db/seeds.rb

```diff
@@ -5,3 +5,6 @@
 #
 #   cities = City.create([{ name: 'Chicago' }, { name: 'Copenhagen' }])
 #   Mayor.create(name: 'Emanuel', city: cities.first)
+#
+users = User.create([{first_name: 'Will', last_name: 'Smith', email: 'will.smith@originate.com'}, {first_name: 'Keanu', last_name: 'Reeves', email: 'keanu.reeves@originate.com'}])
+skills = Skill.create([{name: 'Acting', level: 3, user_id: 1}, {name: 'Acting', level: 1, user_id: 2}])
```

Comment:
Check out [Factory Girl](https://github.com/thoughtbot/factory_girl) It's pretty awesome to easily scaffold out some test data